### PR TITLE
Bug 2116645: Fix Blockpool and Object dashboard for hybrid console

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@bipuladh @SanjalKatiyar this  4.12 fix

File: packages/ocs/block-pool/BlockPoolListPage.tsx [hunk 1]
```diff
@@ -458,15 +458,8 @@ export const BlockPoolListPage: React.FC<BlockPoolListPageProps> = ({}) => {
...
-    blockPoolsLoaded &&
-    (cephLoaded || scLoaded || !compressionLoading || !rawCapLoading);
-  const error =
-    cephError ||
-    scError ||
-    blockPoolsError ||
-    compressionLoadError ||
-    rawCapLoadError;
+  const loaded = blockPoolsLoaded && (cephLoaded || scLoaded);
+  const error = cephError || scError || blockPoolsError;
```
Comment (bipuladh) [inline packages/ocs/block-pool/BlockPoolListPage.tsx:462]: Why are these changes required?
